code/modules/admin/topic.dm

```diff
@@ -1691,3 +1691,9 @@ mob/living/silicon/ai/can_centcom_reply()
 
 	. = "<A HREF='?[source];adminplayerobservejump=\ref[target]'>JMP</A>"
 	. += target.extra_admin_link(source)
+
+/proc/spawn_humanoid_species_admin(var/mob/user, var/mob/M, var/delmob)
+	var/input = input(user, "Select a crew member:") as null|anything in all_species

```

Comment:
Shouldn't this be select a species ?

Reply:
True... some of them are not really a species - like Xenomorphs